test/CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ enable_testing()
 
 set(UMF_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+if(UMF_BUILD_GPU_TESTS AND UMF_BUILD_CUDA_PROVIDER)
+    find_package(CUDA REQUIRED cuda)
+    set(UMF_CUDA_ENABLED TRUE)
+    # TODO do the same for ze_loader
+endif()
```

Comment:
I suppose that this snippet should be moved to the main `CMakeLists.txt`

Reply:
thanks, moved